gr-blocks/include/gnuradio/blocks/add_const_cc.h

```diff
@@ -25,6 +25,7 @@
 
 #include <gnuradio/blocks/api.h>
 #include <gnuradio/sync_block.h>
+#include <volk/volk_alloc.hh>
```

Comment:
Why are they here if they're used in the impl? Was this already missing?

Reply:
@mbr0wn I put that in the public-facing header since it wouldn't build without it and it would be easily visible from the public facing header that the class was using VOLK. I will gladly move it to `add_const_cc_impl.h` and `add_const_ff_impl.h` if you'd like.

Reply:
Hm, I'm OK with that justification.